sqlgen/reflect.go

```diff
@@ -51,16 +51,20 @@ const (
 )
 
 // UnbuildStruct extracts SQL values from a struct
-func UnbuildStruct(table *Table, strct interface{}) []interface{} {
+func UnbuildStruct(table *Table, strct interface{}) ([]interface{}, error) {
```

Comment:
from the commit message:
```
Don't hold onto valuers, instead only use them to get values back. By
doing this, we're allocating the simpler (and more likely to be used)
value onto the heap, while only allocating Valuer onto the heap.
```

do we mean:

```
Don't hold onto valuers, instead only use them to get values back. By
doing this, we're allocating the simpler (and more likely to be used)
value onto the heap, while only allocating Valuer onto the **stack**.
```
?

Reply:
If not maybe we should reword this.

Reply:
Cheers

Reply:
Corrected